src/FhirLoader.Common/BundleClient.cs

```diff
@@ -57,7 +59,7 @@ public async Task Send(ProcessedBundle bundle, Action<int, long>? metricsCallbac
                 _logger.LogTrace($"Sending {bundle.BundleCount} resources to {_client.BaseAddress}...");
 
                 response = await _resiliencyStrategy.ExecuteAsync(
-                    async ct => await _client.PostAsync("", content, ct),
+                    async ct => await _client.PostAsync(requestUri, content, ct),
```

Comment:
This should be POST in most situations, unless the `requestUri` has an `id` at the end. This will allow loading the same profile multiple times against a FHIR service

Reply:
This is post only.
For storing profiles, we are referring to https://learn.microsoft.com/en-us/azure/healthcare-apis/azure-api-for-fhir/store-profiles-in-fhir
This page has section: Accessing profiles and storing profiles
from this section we followed the given sample which has URL with "url" parameter like below for POST:
https://myAzureAPIforFHIR.azurehealthcareapis.com/StructureDefinition?url=http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance

Thats why we used it that way for post.
Now we have changed it to use resource type only like below.
http://whatever/StructureDefinition

Reply:
A customer should be able to run the implementation guide loading multiple times without failure. Also the documentation is wrong because if you are sending a resource to FHIR with an `id`, it generally expected that that will be persisted. Moreover, these resources are standard, and I believe customers would expect them to be at the `id` location as defined in the profile.

FYI @EXPEkesheth - I will file a PR on the docs page for you to look at and approve/deny.